onnxruntime/python/tools/transformers/models/bart/utils/onnx_inference.py

```diff
@@ -0,0 +1,84 @@
+import os
+import time
+
+import numpy as np
+import torch
+from utils import export_helper
+
+from onnxruntime import InferenceSession, SessionOptions
+
+
+def inference(args):
```

Comment:
naming: use actions for functions, e.g. run_inference

Reply:
Done